app/assets/src/components/views/report/TaxonModal.jsx

```diff
@@ -140,82 +136,82 @@ class TaxonModal extends React.Component {
+                {this.state.taxonDescription}
               </div>
-            )}
-            {this.state.taxonParentName && (
-              <div>
-                <div className="taxon-info__subtitle">
-                  Genus: {this.state.taxonParentName}
-                </div>
-                <div className="taxon-info__text">
-                  {this.state.taxonParentDescription}
-                </div>
+            </div>
+          )}
+          {this.state.taxonParentName && (
+            <div>
+              <div className="taxon-info__subtitle">
+                Genus: {this.state.taxonParentName}
               </div>
-            )}
-            {this.state.showHistogram && (
-              <div>
-                <div className="taxon-info__subtitle">
-                  Reference Background: {this.state.background.name}
-                </div>
-                <div
-                  className="taxon-info__histogram"
-                  ref={histogramContainer => {
-                    this.histogramContainer = histogramContainer;
-                  }}
-                />
+              <div className="taxon-info__text">
+                {this.state.taxonParentDescription}
               </div>
-            )}
-            <div className="taxon-info__subtitle">Links</div>
-            <div className="taxon-info__links-section">
-              <ul className="taxon-info__links-list">
-                <li
-                  className="taxon-info__link"
-                  onClick={() => this.linkTo("ncbi")}
-                >
-                  NBCI
-                </li>
-                <li
-                  className="taxon-info__link"
-                  onClick={() => this.linkTo("google")}
-                >
-                  Google
-                </li>
-              </ul>
-              <ul className="taxon-info__links-list">
-                {this.state.wikiUrl && (
-                  <li
-                    className="taxon-info__link"
-                    onClick={() => this.linkTo("wikipedia")}
-                  >
-                    Wikipedia
-                  </li>
-                )}
+            </div>
+          )}
+          {this.state.showHistogram && (
+            <div>
+              <div className="taxon-info__subtitle">
+                Reference Background: {this.state.background.name}
+              </div>
+              <div
+                className="taxon-info__histogram"
+                ref={histogramContainer => {
+                  this.histogramContainer = histogramContainer;
+                }}
+              />
+            </div>
+          )}
+          <div className="taxon-info__subtitle">Links</div>
+          <div className="taxon-info__links-section">
+            <ul className="taxon-info__links-list">
+              <li
+                className="taxon-info__link"
+                onClick={() => this.linkTo("ncbi")}
+              >
+                NBCI
```

Comment:
You might run into conflicts here once you merge master back... `NBCI` -> `NCBI`